### DISCUSSION TITLE
tokenizer

### DISCUSSION THREAD

**Discussion by baseplate-admin** [flet-dev/flet#3744] [Packages] [answered]
Add support for `tokenizer`

- [Github](https://github.com/huggingface/tokenizers)
- [Pypi](https://pypi.org/project/tokenizers/)

Note:
- Uses pyo3

**Comment by ndonkoHenri** [comment, marked as answer]
Tokenizers 0.22.2 is supported on Android and iOS: https://pypi.flet.dev/tokenizers

0.23.1 (the latest) will be supported as from tomorrow.
